src/components/Products/Products.jsx

```diff
@@ -0,0 +1,194 @@
+import "./products.css";
+
+const Products = () => {
+    return (
+        <main className="product_container">
+            <div className="product_heading flex flex_justify_start flex_align_center">
+                <h3>Showing All Products</h3>
+                <p>(Showing 10 Products)</p>
+            </div>
+
+            <div className="product_list">
+
+                <div className="card_image flex">
```

Comment:
I know you are making static pages here but you could have also directly used data from DB to map here.